scripts/check_release_version.py

```
@@ -0,0 +1,52 @@
+import re
+from pathlib import Path
+
+from packaging import version
+
+from git import Repo
+import toml
+
+
+def get_git_version():
+    repo = Repo()
+    assert not repo.bare
+    tag_strings = [t.name for t in repo.tags]
+    tag_strings.sort(reverse=True)
+
+    latest_tag = tag_strings[0].strip()
+    assert len(latest_tag) > 0
```

Rather throw an exception indicating what exactly is wrong, e.g.

```python
if not latest_tag:
    raise Exception("Repository does not have any git tags.")
```